ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master__okd-scos.yaml

```diff
@@ -0,0 +1,45 @@
+base_images:
+  ocp_4.16_base-rhel9:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-openshift-4.16
+  origin_scos-4.16_base-stream9:
+    name: scos-4.16
+    namespace: origin
+    tag: base-stream9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+    origin_scos-4.16_base-stream9:
+      as:
+      - registry.ci.openshift.org/origin/4.16:base-stream9
+  to: ptp
+promotion:
+  to:
+  - name: ptp-4.17
```

Comment:
```suggestion
  - name: ptp-4.16
```

Reply:
replied on the other thread

Reply:
@nishant-parekh apologies, I meant to say master is still 4.16 at this time, it has not been branched from master yet. This should be ptp-4.16 as suggested imo.